flask_jwt_extended/config.py

```diff
@@ -255,6 +255,10 @@ def user_claims_in_refresh_token(self):
     def exempt_methods(self):
         return {"OPTIONS"}
 
+    @property
+    def error_msg_key(self):
+        return current_app.config.get('JWT_ERROR_MESSAGE_KEY', 'msg')
```

Comment:
Just to stay consistent with the rest of the app here, could we remove this `.get()` and have it just access the `JWT_ERROR_MESSAGE_KEY` directly, and add this option as a `app.config.setdefault` in the JWTManager?